Sources/Target/Windows/Process.cpp

```diff
@@ -105,15 +122,7 @@ ErrorCode Process::detach() {
   return kSuccess;
 }
 
-#if defined(ARCH_X86)
-static uint8_t const debugBreakCode[] = {
-    0xCC,                         // 00: int 3
-    0xB8, 0x00, 0x00, 0x00, 0x00, // 01: mov eax, RtlExitUserThread address
-    0xFF, 0xD0                    // 06: call eax
-};
-
 BOOL Process::writeDebugBreakCode(uint64_t address) {
```

Comment:
I like how this function became 100% architecture agnostic, and the `#ifdef` is limited to `debugBreakCode`.